### DISCUSSION TITLE
Is it possible to open/ read other file formts like a txt-file

### DISCUSSION THREAD

**Discussion by him2013** [yasirkula/UnityNativeGallery#211] [General]
Hi,
at first thank you very much for this super job.
I'm trying to select a text file. But only image, video and audio files are active, all others are deactivated. I would like to only get the url of the selected file back? I would be happy if you can tell me how I can adapt the source code.
thanks

**Comment by yasirkula** [comment]
You can use NativeFilePicker: https://github.com/yasirkula/UnityNativeFilePicker

**Reply by him2013** [reply]
waw. works perfect. thank you very much.
